src/future-hopr-lib-components/NFT/nftImageInteractive.jsx

```diff
@@ -167,18 +167,9 @@ export default function NftImageInteractive(props) {
           set_rotated(false);
         }}
       >
-        <div
-          className={[
-            'nft-front',
-            fliped ? 'flip' : '',
-            rotated ? 'rotate' : '',
-          ].join(' ')}
-        >
+        <div className={['nft-front', fliped ? 'flip' : '', rotated ? 'rotate' : ''].join(' ')}>
           <div
-            className={[
-              'nft-image-container',
-              props.ignored ? 'ignored' : '',
-            ].join(' ')}
+            className={['nft-image-container', props.ignored ? 'ignored' : ''].join(' ')}
```

Comment:
@camilovegag, please check if we can make an exception for arrays in props to have singleAttributePerLine